src/cider/nrepl/middleware/ns.clj

```diff
@@ -100,7 +101,13 @@
   {:ns-vars-with-meta (ns-vars-with-meta msg)})
 
 (defn- ns-path-reply [msg]
-  {:path (ns-path msg)})
+  (let [v (ns-path msg)
+        cljs? (cljs/grab-cljs-env msg)]
+    {:path v
+     :url (if cljs?
```

Comment:
Shouldn't we return a URL for Clojure as well?

Reply:
For Clojure they already are URLs

Reply:
Hmm, so why not just put the URL for ClojureScript into the `file` key then? For me it just looks weird when the return structure differ based on the runtime. Alternatively you can stick the Clojure `path` under `url` as well, so at least it'd be consistent.

Reply:
The return structure is the same for both runtimes: `:path`, `:url`.

`:path` remains unchanged.

`:url` equals `:path` on the JVM, because `:path` always has been a URL for JVM Clojure.

`:url` is different from `:path` in Clojurescript. Ideally, `:path` would have been a URL to begin with.

This is the best backwards-compatible change I could think of.

Reply:
Ah, I didn't read this carefully.